hnn_core/network.py

```diff
@@ -1428,6 +1431,76 @@ def add_electrode_array(self, name, electrode_pos, *, conductivity=0.3,
                                      method=method,
                                      min_distance=min_distance)})
 
+    def update_weights(self, e_e=None, e_i=None,
+                       i_e=None, i_i=None, copy=False):
```

Comment:
something I realized late ... might be worth adding this to an example to show off that it's possible to do. Otherwise it's kind of hidden from the user.